babel.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
I had to rename the file to bypass a breaking change in babel 7 https://github.com/facebook/jest/issues/6053#issuecomment-383632515